drivers/storage/vbox/storage/vbox_storage.go

```diff
@@ -674,7 +675,7 @@ func (d *driver) attachVolume(
 		return err
 	}
 
-	if err := m.Refresh(); err != nil {
+	if err = m.Refresh(); err != nil {
```

Comment:
Hi @codenrhoden,

I have a question. Here, and the other locations in this file, why did you switch to re-using the previously declared `err` var? I see no reason to do so, and keeping  it local to the scope of the if blocks keep it scoped to a specific usage, no?